types/google-protobuf/google-protobuf-tests.ts

```diff
@@ -38,6 +38,7 @@ import * as google_protobuf_wrappers_pb from "google-protobuf/google/protobuf/wr
    google.protobuf.Timestamp some_timestamp = 13;
    google.protobuf.Type some_type = 14;
    google.protobuf.DoubleValue some_double_value = 15;
+   map<int32, string> some_key_value = 16;
```

Comment:
`some_key_value_map`?

Reply:
`Map` is automatically added by protoc at the end of the field name like `List` on `repeated` fields.

Reply:
Interesting.